lib/usr/clib/msgchan/msgchan.h

```diff
@@ -25,11 +25,28 @@
 extern "C" {
 #endif
 
+#ifndef MC_NAME_SIZE
```

Comment:
I'm assuming this is because the "priv" file needs this definition too? The duplication is not ideal, could the "priv" header just include this one? msgchan.c includes both anyway so there's no benefit for compilation time or anything.

Reply:
fixed